keyboards/coban/pad3a/rules.mk

```diff
@@ -0,0 +1,4 @@
+MOUSEKEY_ENABLE = no
+EXTRAKEY_ENABLE = yes
+ENCODER_ENABLE = yes
+ENCODER_MAP_ENABLE = yes
```

Comment:
Encoder map shouldn't be enabled at the keyboard level — this belongs in `keymaps/*/rules.mk`.
```suggestion
```